en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md

```diff
@@ -0,0 +1,207 @@
+# Configure Multiple Gateways with Choreo Connect
+
+Choreo Connect supports the configuration of multiple gateways with APIM or stand-alone mode. Basically, there are two types of deployments to setup multiple gateways.
```

Comment:
```suggestion
Choreo Connect supports the configuration of multiple Gateways with API Manager or in stand-alone mode. There are two types of deployments to setup multiple Gateways.
```

Let's use title case for Gateway's and other components everywhere.

Reply:
Done with https://github.com/wso2/docs-apim/pull/3839/commits/92c57f77931f5c5ba22596f3c0fb108093b38d55